build.sbt

```diff
@@ -941,24 +941,15 @@ lazy val releaseSettings = Seq(
       case _ => Seq[ReleaseStep]()
     }
   },
-  pomExtra := (
-    <url>http://github.com/getquill/quill</url>
-    <licenses>
-      <license>
-        <name>Apache License 2.0</name>
-        <url>https://raw.githubusercontent.com/getquill/quill/master/LICENSE.txt</url>
-        <distribution>repo</distribution>
-      </license>
-    </licenses>
-    <scm>
-      <url>git@github.com:getquill/quill.git</url>
-      <connection>scm:git:git@github.com:getquill/quill.git</connection>
-    </scm>
-    <developers>
-      <developer>
-        <id>fwbrasil</id>
-        <name>Flavio W. Brasil</name>
-        <url>http://github.com/fwbrasil/</url>
-      </developer>
-    </developers>)
+
+  homepage := Some(url("http://github.com/getquill/quill")),
+  licenses := List(("Apache License 2.0", url("https://raw.githubusercontent.com/getquill/quill/master/LICENSE.txt"))),
+  developers := List(
+    Developer("fwbrasil", "Flavio W. Brasil", "", url("http://github.com/fwbrasil")),
+    Developer("deusaquilus", "Alexander Ioffe", "", url("https://github.com/deusaquilus"))
+  ),
+  scmInfo := Some(
+    ScmInfo(url("https://github.com/getquill/quill"), "git:git@github.com:getquill/quill.git")
```

Comment:
The first entry is meant to be a valid URL to where the source code is hosted and the second entry is meant to be the main retrieval URL for the source (note that theoretically speaking this can also be the same as the homepage but most people use the `git:` handle for cloning source code from github)